backend/core/views.py

```
@@ -29,38 +50,134 @@ def create(self, request, *args, **kwargs):
+                'user' : profile_serializer.data['user']
+            }
             
-            return Response(profile_serializer.data['user'], status=status.HTTP_201_CREATED)
+            return Response(token_data, status=status.HTTP_201_CREATED)
         
         return Response(profile_serializer.errors, status=status.HTTP_400_BAD_REQUEST)
 
-class RandomCard(APIView):
+class Login(APIView):
+    def post(self, request, *args, **kwargs):
+        email = request.data.get("email")
```

Ruff

Local variable `email` is assigned to but never used

Remove assignment to unused variable `email`

(F841)